.github/workflows/landblocking_tcb.yml

```diff
@@ -0,0 +1,244 @@
+                        if span["line_start"] >= lines[0] and span["line_end"] <= lines[1]:
+                          print("lines hit for", span["file_name"])
+                          lint_lines_found.write(message)
+                          break
+              except:
+                pass
+
+            # reset clippy output for next pass
+            clippy_output.seek(0)
+          END
+
+      - name: Display lint for file
+        if: success()
+        run: |
+          cat lint_found
+
+      - name: Display lint for file + lines
+        if: success()
+        run: |
+          cat lint_lines_found
+
+      - name: Write comment on PR if lints are found
+        if: success()
+        uses: actions/github-script@0.5.0
+        env:
+          PULL_ID: ${{ github.event.client_payload.pull_id }}
+          OWNER: ${{ github.event.client_payload.owner }}
+        with:
+          github-token: ${{secrets.MIRAI_BOT}}
+          script: |
+            const fs = require('fs');
+            var path = require('path');
+
+            async function main() {
+              // get lints
+              const lint_found = fs.readFileSync('lint_found');
+              const lint_lines_found = fs.readFileSync('lint_lines_found');
+
+              // don't post anything if relevant files are not hit
+              if (lint_lines_found.length < 5) { // empty hack
+                return;
+              }
+
+              // create the message
+              var message = `Hello you!
+
+            It looks like #clippy found some allowed lints that might be useful to you:
+
+            <details><summary>click here to reveal them</summary>
+            <pre><code>${lint_lines_found}</code></pre>
+            </details>
+
+            If you want these helpful messages on your PRs too, [ask here](https://github.com/mimoo/mirai-bot/issues).
+            `;
+
+              // delete previous comments from MIRAI-BOT that includes the tag "#clippy"
+              const comments = await github.issues.listComments({
+                owner: "libra",
+                repo: "libra",
+                issue_number: process.env.PULL_ID
+              });
+              console.log("comments", comments);
+              for (let comment of comments.data) {
+                console.log("comment", comment);
+                console.log("comment.user.login", comment.user.login);
+                console.log(comment.user.login === "MIRAI-bot");
+                console.log("comment.body", comment.body);
+                console.log(comment.body.includes("#clippy"));
+                if (comment.user.login === "MIRAI-bot" && comment.body.includes("#clippy")) {
+                  await github.issues.deleteComment({
+                    owner: "libra",
+                    repo: "libra",
+                    comment_id: comment.id
+                  });
+                }
+              }
+
+              // comment
+              await github.issues.createComment({
```

Comment:
@mimoo  here, instead of creating a comment, I basically want to block the PR unless the issue is fixed. any ideas on how to do that?